docs/overview.md

```diff
@@ -151,6 +151,20 @@ If any change in logic is needed, the Executor/s should be migrated to a new one
 
 <br>
 
+- [PermissionedPayloadsController](../src/contracts/payloads/PermissionedPayloadsController.sol): permissioned version of the payloads controller.
```

Comment:
Still missing explanation on what this PayloadsController will be used for. (conceptually)